poetry.lock

```diff
@@ -534,11 +545,11 @@ python-versions = "*"
 
 [[package]]
 name = "pycodestyle"
-version = "2.8.0"
+version = "2.9.1"
 description = "Python style guide checker"
 category = "main"
 optional = false
-python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
+python-versions = ">=3.6"
```

Comment:
We're going to have to wait before merging because of this line see https://github.com/PyCQA/prospector/pull/511#issuecomment-1156178453.

Reply:
```suggestion
python-versions = ">=3.7"
```

Reply:
I don’t think I follow. pycodestyle 2.9.1 still supports python 3.6, I see no reason why this change should be considered one that breaks python 3.6 compatibility.

Reply:
I think the current error is due to the lack of support to python 3.6 in one of the dependencies (don't know which one):
```
       Complete output (6 lines):
      Traceback (most recent call last):
        File "<string>", line 1, in <module>
        File "/home/runner/.cache/pre-commit/repof1w7zove/setup.py", line 1
          from __future__ import annotations
```
``from __future__ import annotations`` requires python 3.7 afaik.

Reply:
I pushed a new commit where I ran `poetry update mccabe --lock` from a python 3.6 env and pushed that. Then I was able to `pip install .` from my branch on a python 3.6 env and everything installed fine.

```
Successfully installed PyYAML-6.0 astroid-2.11.7 dill-0.3.4 dodgy-0.2.1 flake8-5.0.4 flake8-polyfill-1.0.2 importlib-metadata-4.2.0 isort-5.10.1 lazy-object-proxy-1.7.1 mccabe-0.7.0 pep8-naming-0.10.0 platformdirs-2.4.0 prospector-1.7.7 pycodestyle-2.9.1 pydocstyle-6.1.1 pyflakes-2.5.0 pylint-2.13.9 pylint-celery-0.3 pylint-django-2.5.3 pylint-flask-0.6 pylint-plugin-utils-0.7 requirements-detector-0.7 setoptconf-tmp-0.3.1 snowballstemmer-2.2.0 toml-0.10.2 tomli-1.2.3 typed-ast-1.5.4 typing-extensions-4.1.1 wrapt-1.14.1 zipp-3.6.0
```
Including `flake8-5.0.4`, `mccabe-0.7.0`, and `pycodestyle-2.9.1`.

Reply:
I was hoping to show that that the changes in this PR are unrelated to the workflow failure by opening a dummy PR, https://github.com/PyCQA/prospector/pull/517, that only changes the readme. I guess it won't actually run unless it's approved.